src/content/docs/nrql/nrql-examples/browserspa-nrql-query-examples.mdx

```diff
@@ -17,6 +17,7 @@ redirects:
   - /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/new-relic-browserspa-nrql-query-examples
   - /docs/query-spa-data-insights
   - /docs/browser/single-page-app-monitoring/use-spa-data/query-spa-data-new-relic
+  - /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
```

Comment:
I don't think this is in the yml file. when I click it in the nav, it disappears